eng/Versions.props

```diff
@@ -21,5 +21,7 @@
     <MicrosoftNETCoreILDAsmVersion>6.0.0-preview.6.21352.12</MicrosoftNETCoreILDAsmVersion>
     <!-- SDK dependencies -->
     <MicrosoftDotNetGenAPITaskPackageVersion>9.0.100-alpha.1.23618.1</MicrosoftDotNetGenAPITaskPackageVersion>
+    <!-- xUnit dependencies -->
+    <XunitSkippableFactVersion>1.4.13</XunitSkippableFactVersion>
```

Comment:
It's absolutely fine to use SkippableFact here. Just wanted to let you know that we already have in-built support for this and more in Arcade via [Microsoft.DotNet.XUnitExtensions](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.XUnitExtensions).

Reply:
This is great to know! Thanks for pointing this out.